utils/swift_build_support/swift_build_support/products/tsan_libdispatch.py

```diff
@@ -98,7 +97,6 @@ def get_dependencies(cls):
         return [cmark.CMark,
                 llvm.LLVM,
                 libcxx.LibCXX,
-                libicu.LibICU,
```

Comment:
Hold up, I made a mistake: this still needs libicu because it builds Foundation. I'll push a fix.

Reply:
Fixed now.